framework/resources/Functional/parquet_storage/parquet_autoPrtn/autoPrtnPrqNstd.json

```diff
@@ -1,26 +1,27 @@
 {
-    "testId": "parquet_autoPrtn",
-    "type": "group",
-    "description": "Query auto partitioned parquet data that is stored in nested direcotries, where directory name is greater than 8 characters",
+    "testId": "parquet_autoPrtn", 
```

Comment:
why the extra spaces at the end? Here and in many other places. The only change should be the word "parquet" added in categories. It'll make review simpler. Please go through all files and update.